miso-web/src/main/webapp/WEB-INF/event-config.xml

```diff
@@ -28,6 +28,17 @@
        http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.1.xsd http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd"
   default-autowire="byName">
 
+  <bean name="prometheusInterceptor" class="uk.ac.bbsrc.tgac.miso.webapp.context.PrometheusInterceptor" />
+   <bean id="prometheusAdvisor"
+      class="org.springframework.aop.support.RegexpMethodPointcutAdvisor">
+      <property name="advice">
+         <ref local="prometheusInterceptor"/>
+      </property>
+      <property name="pattern">
+         <value>uk\.ac\.bbsrc\.tgac\.miso\.(sqlstore\.SQL|persistence\.impl\.Hibernate).*</value>
```

Comment:
Yes! I had it in before when it wasn't working, but it no longer seems necessary.